subt/zmq-subt-x4.json

```diff
@@ -66,11 +66,24 @@
               "camera_xyz": [0.1, 0.0, -0.08],
               "image_size": [640, 480],
               "principal_point": [320.5, 240.5],
+              "lidar_fov": 6.283185307179586,
+              "min_z": -0.2,
+              "max_z": 0.4,
+              "max_slope": -3.14,
               "disabled_vertical_diff_limit": 0.698, "vd_comment": "radians(40)",
               "max_slope": -0.436, "ms_comment": "radians(-25)"
             }
           }
       },
+      "vo": {
+        "driver": "subt.visual_odometry:Localization",
+        "init": {
+          "xyz": [0.1, 0, -0.08],
+          "rpy": [0.0, 0.0, 0.0],
+          "principal-point": [320.5, 180.5],
```

Comment:
I think this is not correct. See
https://github.com/osrf/subt/blob/60aeb723be6af834c1499221f2a7ad16f926321c/submitted_models/ssci_x4_sensor_config_2/model.sdf#L311-L353
The camera for `ssci_x4_sensor_config_2` is 640x480.

Reply:
yes, I can confirm the resolution (it is BTW the only output I get from `detector.stdout` = no detections). I will fix it, thanks.

Reply:
do we know if also `"focal-length": 554.25469` is correct/wrong?

Reply:
Thanks for the hint with link to the source:
```
<fx>554.25469</fx>
```
so this part is OK.